tests/subsurfaces.cpp

```diff
@@ -470,6 +470,36 @@ TEST_P(SubsurfaceTest, one_subsurface_to_another_fallthrough)
                     wl_fixed_from_int(pointer_y_2 - subsurface_top_y))));
 }
 
+TEST_P(SubsurfaceTest, place_below_simple)
+{
+    auto subsurface_moving_down{wlcs::Subsurface::create_visible(main_surface, 0, 0, subsurface_width, subsurface_height)};
+    wl_subsurface_place_below(subsurface_moving_down, subsurface);
```

Comment:
This is actually double-buffered state, so needs a `wl_surface.commit()` on the parent before it'll be applied.

Reply:
Fixed